src/en/NMTranslations.lua

```diff
@@ -16,11 +16,13 @@ return {
 			local doc = GETDocument(baseURL)
 
 			return map(doc:select("div.p-4"), function(v)
-				local title = v:selectFirst("h2.mb-3"):selectFirst("a")
+				local title = v:selectFirst("h2.text-center"):text()
+				local link = v:selectFirst("a"):attr("href")
+
 				return Novel {
-					title = title:text(),
-					link = title:attr("href"),
-					imageURL = baseURL .. v:selectFirst("img.object-cover"):attr("src"),
+					title = title,
+					link = link,
```

Comment:
I think this is bad style to create a local variable that is only used once. This likely creates the variable and then copies it when creating the Novel object.

Reply:
It would not have a significant runtime impact, but it is indeed bad style.